### ISSUE TITLE
[Suggestion] Support for low resolution without them looking blurry

### ISSUE THREAD

**Issue by MrJarW** [nukeykt/NBlood#400]
I understand the reason why when lowering the resolution will blur out everything, so my suggestion is that NBlood always sets the game's internal resolution to the monitor's native, that way when you lower it down in-game, it can be more accurate, and it can support a stable version of, for example, 320x200 in a higher display.
I know ports and backports that do this, like GZDoom, the upcoming Forces Engine backport, basically all ports based on Chocolate Doom, among some others.
This will greatly boost the accuracy NBlood is trying to go for, and in some way or another, it's already in the game, like when the player gets the delirium shroom.

**Comment by MaxiM-PL** [comment]
Agreed. Daggerfall Unity developer also did that some time ago as per request and it definitely works in the game's favor. Now you can play it in 320x200 and 640x400 (with aspect correction ofc) on any screen with crispy-sharp pixels ;)

**Comment by CommonLoon102** [comment]
The software (classic) renderer has the `r_upscalefactor` command which does what you want. But it is not available for polymost (OpenGL) though.
